azure_pipeline.yaml

```diff
@@ -235,6 +254,7 @@ parameters:
       storage_account_rg: 'core-infra-stg-rg'
       storage_account_name: 'cftappsstg'
       dependsOn: 'sbox_apim'
+      pipeline_tests: true
```

Comment:
this needs a specific test for APIM, we haven't integrated the apps in APIM yet so this can wait

(check roadmap for APIM strawberry)